src/core/utils.js

```diff
@@ -8,3 +10,11 @@ export function ngettext(singular, plural, n) {
   }
   return plural;
 }
+
+export function camelCaseProps(obj) {
```

Comment:
Sounds good - when I get to setting up the store properly I'll look at changing that over.